Codeinwp/neve
# Fix ordering react controls

### Summary
Prevent rendering elements that are in the database but shouldn't be displayed.

### Will affect visual aspect of the product
NO

### Test instructions
This happens for both meta ordering control and the ordering control in the customizer.

To test the meta control:
1. Enable all elements in a post's dashboard ( in the single post editor )
2. Go to the customizer, switch to cover header for posts
3. Go back to single post settings in the editor. With this fix, there should not be empty elements.

To test the customizer control:
Follow the same steps from the issue's text. It should not be present anymore.



Closes Codeinwp/neve-pro-addon#1707.
